app/tests/test_views.py

```diff
@@ -299,6 +299,74 @@ def test_step_3_with_invalid_areaid_is_not_found(self):
         self.assert404(self.client.get('/my-expertise/blah'))
         self.assert404(self.client.get('/my-expertise/blah/1'))
 
+class MatchMeTests(ViewTestCase):
+    def setUp(self):
+        super(MatchMeTests, self).setUp()
+        self.login()
+
+        learn = LEVELS['LEVEL_I_WANT_TO_LEARN']['score']
+        refer = LEVELS['LEVEL_I_CAN_REFER']['score']
+        explain = LEVELS['LEVEL_I_CAN_EXPLAIN']['score']
+        do_it = LEVELS['LEVEL_I_CAN_DO_IT']['score']
+
+        skill = "opendata-open-data-policy-core-mission"
+
+        self.last_created_user.set_skill(skill, learn)
+
+        UserFactory.create(
+            first_name=u"connector",
+            last_name=u"stone",
+            connections=[],
```

Comment:
An `IntegrityError` like this is raised if `connections=[]` isn't in here:

```
(sqlite3.IntegrityError) NOT NULL constraint failed: user_skills.user_id [SQL: u'INSERT INTO user_skills (created_at, updated_at, level, name, user_id) VALUES (?, ?, ?, ?, ?)'] [parameters: ('2015-10-15 14:48:09.279803', '2015-10-15 14:48:09.279810', 1, 'opendata-open-data-policy-core-mission', None)]
```

That's unfortunate, since it makes things unnecessarily a bit more verbose...